README.md

```diff
@@ -1,4 +1,6 @@
-**Project move to [SourceForge](https://sourceforge.net/p/dex2jar) and [Bitbucket](https://bitbucket.org/pxb1988/dex2jar) and [Github](https://github.com/pxb1988/dex2jar)**
+# dex2jar 

```

Comment:
Please remove the trailing space character on this line.

Reply:
Trailing space removed 👍